Unity/Assets/Scripts/StartButtonScript.cs

```diff
@@ -1,78 +1,64 @@
-using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
-using UnityEngine.Events;
 
+/// <summary>
+/// 
+/// </summary>
 public class StartButtonScript : MonoBehaviour
 {   
-    public GameObject MainGameObject;
-    public GameObject BattleGameObject;
-
-    public GameObject NextGameButton;
-
-    public Image creatureImage1;
-    public Image creatureImage2;
-
-    public Image creatureBattleImage1;
-    public Image creatureBattleImage2;
-
-    public Image VSImage;
-
-    private Image fieldImage;
-
-    public Sprite TokyoImage;
-    public Sprite SavannahImage;
-    public Sprite PacificOceanImage;
-    public Sprite SpaceImage;
-
-    public Sprite ElephantImage;
-    public Sprite LionImage;
-    public Sprite ZebraImage;
-    public Sprite DolphinImage;
-    public Sprite OrcaImage;
-    public Sprite HumanImage;
-
-    public Text creature1Text;
-    public Text creature2Text;
-
-    public Text BattleText;
-    public Image BattleTextImage;
-
-    private int creature1_number;
-    private int creature2_number;
-
-    private int c1_scaled_power;
-    private int c2_scaled_power;
-
-    private string field;
+    const string _savannah = "サバンナ";
+    const string _pacificOcean = "太平洋";
+    const string _tokyo = "東京";
+
+    [SerializeField] GameObject _mainGameObject;
+    [SerializeField] GameObject _battleGameObject;
+    [SerializeField] GameObject _nextGameButton;
+
+    [SerializeField] Image _creatureImagePlayer;
+    [SerializeField] Image _creatureImageOpponent;
+    [SerializeField] Image _creatureBattleImagePlayer;
+    [SerializeField] Image _creatureBattleImageOpponent;
+    [SerializeField] Image _vsImage;
+    [SerializeField] Image _battleTextImage;
+    [SerializeField] Image _fieldImage;
+
+    [SerializeField] Text _creatureTextPlayer;
+    [SerializeField] Text _creatureTextOpponent;
+    [SerializeField] Text _battleText;
+
+    [SerializeField] List<Sprite> _fieldImages;
+    [SerializeField] List<Sprite> _characterImages;
+
+    int _creatureNumberPlayer;
+    int _creatureNumberOpponent;
```

Comment:
単語の最後は型がわかる名前だと良いです。
あと、英語としておかしな順番になっていますね。
```suggestion
    int _playerCreatureNumber;
    int _opponentCreatureNumber;
```